### PR TITLE
GApps

### PR DESCRIPTION
**Pull request by JonnyVR1**

### CODE REVIEW

**Comment by phhusson** [comment]
We're not going to use a manifest not hosted in a TrebleDroid git

**Comment by JonnyVR1** [comment]
Apologies, already fixed - leftover from testing

File: build.sh [hunk 1]
```diff
@@ -15,16 +15,16 @@ aosp="android-13.0.0_r49"
...
-manifest_url="https://android.googlesource.com/platform/manifest"
 
 repo init -u "$manifest_url" -b $aosp --depth=1
 if [ -d .repo/local_manifests ] ;then
 	( cd .repo/local_manifests; git fetch; git reset --hard; git checkout origin/$phh)
 else
 	git clone https://github.com/TrebleDroid/treble_manifest .repo/local_manifests -b $phh
 fi
-repo sync -c -j 1 --force-sync || repo sync -c -j1 --force-sync
+repo sync -c -j8 --force-sync || repo sync -c -j8 --force-sync
```
Comment (JonnyVR1) [inline build.sh:25]: Fixes issues running in a WSL environment

File: build.sh [hunk 2]
```diff
@@ -42,14 +42,21 @@ bash "$originFolder"/list-patches.sh
...
+		git clone https://github.com/phhusson/vendor_vndk -b android-10.0
+	fi
 )
 
-buildVariant treble_arm64_bvS-userdebug td-arm64-ab-vanilla
-( cd sas-creator; bash lite-adapter.sh 64; xz -c s.img -T0 > ../release/$rom_fp/system-td-arm64-ab-vndklite-vanilla.img.xz )
-
-buildVariant treble_a64_bvS-userdebug td-arm32_binder64-ab-vanilla
-( cd sas-creator; bash lite-adapter.sh 32; xz -c s.img -T0 > ../release/$rom_fp/system-td-arm32_binder64-ab-vndklite-vanilla.img.xz )
+buildVariant treble_arm64_bgN-userdebug td-arm64-ab-gapps
```
Comment (JonnyVR1) [inline build.sh:59]: Add secure build variants to the CI builds

File: build_dirty.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```
Comment (JonnyVR1) [inline build_dirty.sh:1]: Add ability to ignore repo sync as a dirty build - useful for debugging

**Pull request closed without merging**